src/components/accession2/view/Accession2View.tsx

```diff
@@ -612,11 +617,7 @@ export default function Accession2View(): JSX.Element {
           />
         </Grid>
         <Grid item xs={getOverviewGridSize(2)}>
-          <OverviewItemCard
-            isEditable={false}
-            title={strings.AGE}
-            contents={accession?.collectedDate ? `${age.value} ${age.unit}` : null}
-          />
+          <OverviewItemCard isEditable={false} title={strings.AGE} contents={age ? age : null} />
```

Comment:
an alternative to this ternary is `contents={age || null}` .. in case you prefer that